AUTHORS

```diff
@@ -877,7 +877,8 @@ Martin von Gagern <Martin.vGagern@gmx.net>
 Hideki Yamamura <hideki.yamamura@gmail.com>
 Lenny Markus <lmarkus@paypal.com>
 Nelson Pecora <nelson@yoshokatana.com>
-Graham Fairweather <xotic750@gmail.com>
+ronkorving <ron@ronkorving.nl>
+Xotic750 <xotic750@gmail>
 Nicholas Young <nicholas@nicholaswyoung.com>
 chrisjohn404 <chris.s.johnson4@gmail.com>
```

Comment:
Should be `Chris Johnson` per https://github.com/nodejs/node/pull/6373#discussion_r61674301